roles/core/precheck/defaults/main.yml

```diff
@@ -49,6 +49,10 @@ scale_prepare_disable_selinux: false
 ## rules prior to running this role (e.g. as pre_tasks)
 scale_prepare_disable_firewall: false
 
+## For the Spectrum Scale daemon network the 'ListenAddress' needs to be set
+## at /etc/ssh/sshd_config to allow only the admin IP to login.
+## Set the scale_sshd_config_update to 'true' to update the sshd_config
+scale_sshd_config_update: false
```

Comment:
With the current code one needs to enable `scale_prepare_enable_ssh_login` AND `scale_sshd_config_update` to enable the task - is this on purpose? If this is on purpose then we should document this dependency.

Also, the name `scale_sshd_config_update` is quite generic - in fact this variable only controls the `ListenAddress` task - should this be reflected in it's name?